draft-ietf-mls-protocol.md

```diff
@@ -886,20 +890,20 @@ encrypted secret values are computed as follows:
   * Nonce: The nonce derived from Z
   * Additional Authenticated Data: The empty octet string
   * Plaintext: The secret value, without any further formatting
-* Encode the ECIESCiphertext with the following values:
+* Encode the HPKECiphertext with the following values:
   * ephemeral\_key: The ephemeral public key x\*G
   * ciphertext: The AEAD output
 
 ~~~~~
-key = HKDF-Expand(Secret, ECIESLabel("key"), Length)
-nonce = HKDF-Expand(Secret, ECIESLabel("nonce"), Length)
+key = HKDF-Expand(Secret, HPKELabel("key"), Length)
+nonce = HKDF-Expand(Secret, HPKELabel("nonce"), Length)
 
-Where ECIESLabel is specified as:
+Where HPKELabel is specified as:
 
 struct {
   uint16 length = Length;
   opaque label<12..255> = "mls10 ecies " + Label;
-} ECIESLabel;
+} HPKELabel;
```

Comment:
I'm inclined to go ahead and change this out for a reference to the CFRG HPKE draft.  If that doesn't get adopted by CFRG for some reason, we can always revert.

Reply:
Ok, I removed the description and added an informative reference to HPKE.